ui/src/app/workflows/components/workflow-creator.tsx

```diff
@@ -93,9 +93,9 @@ export const WorkflowCreator = ({namespace, onCreate}: {namespace: string; onCre
                         entrypoints={(workflowTemplate.spec.templates || []).map(t => t.name)}
                         parameters={workflowTemplate.spec.arguments.parameters || []}
                     />
-                    <Button outline={true} onClick={() => setStage('full-editor')}>
-                        Edit using full workflow options
-                    </Button>
+                    <a onClick={() => setStage('full-editor')}>
```

Comment:
![image](https://user-images.githubusercontent.com/1142830/105527820-0810c300-5c99-11eb-9267-594abda450b2.png)

Reply:
changed for consistency